domain/secretbackend/state/types.go

```diff
@@ -242,3 +244,10 @@ type CloudRow struct {
 	// ModelOwnerUserName holds the name of the user who owns the model.
 	ModelOwnerUserName string `db:"model_owner_user_name"`
 }
+
+// SecretBackendInfo represents a secret backend.
+type SecretBackendInfo struct {
+	BackendUUID string `db:"uuid"`
+	Name        string `db:"name"`
```

Comment:
This should be SecretBackendName to avoid confusion since the other similar struct uses Name for modle name.
Also doesn't need a db tag (just used as a query param).
I don't think any of the struct attributes need db tags, since they are all explicitly referenced, right?